application/common/dynamodb.py

```diff
@@ -20,6 +20,47 @@ def dict_hash(change_event: Dict[str, Any], sequence_number: str) -> str:
     return change_event_hash.hexdigest()
 
 
+def put_circuit_is_open(circuit: str, is_open: bool) -> None:
+    dynamo_record = {
+        "Id": circuit,
+        "ODSCode": "CIRCUIT",
+        "IsOpen": is_open,
+    }
+    try:
+        dynamodb = boto3.client("dynamodb", region_name=environ["AWS_REGION"])
+        serializer = TypeSerializer()
+        put_item = {k: serializer.serialize(v) for k, v in dynamo_record.items()}
+        response = dynamodb.put_item(TableName=environ["CHANGE_EVENTS_TABLE_NAME"], Item=put_item)
+        logger.info("Put circuit status", extra={"response": response, "item": put_item})
+    except Exception as err:
+        logger.exception(f"Unable to insert a record into dynamodb.Error: {err}")
+        raise
+
+
+def get_circuit_is_open(circuit: str) -> Union[bool, None]:
```

Comment:
Function could do with a description block like "add_change_request_to_dynamodb", without context not sure what the purpose of getting this open stat is for